src/main/java/org/sonatype/nexus/itscriptclient/blobs/BlobStore.java

```diff
@@ -0,0 +1,41 @@
+package org.sonatype.nexus.itscriptclient.blobs;
+
+import org.sonatype.nexus.itscriptclient.script.ScriptResult;
+import org.sonatype.nexus.itscriptclient.script.ScriptRunner;
+
+public class BlobStore
+{
+  static final String CREATE_BLOB = "import org.sonatype.nexus.blobstore.api.BlobStore\n" +
```

Comment:
I wonder if there is a better way for us to store these scripts? No changes are required here but we should think it through.

Reply:
Probably best to do it in groovy usinghttp://docs.groovy-lang.org/latest/html/documentation/index.html#_triple_double_quoted_string, it would at least be cleaner looking if thats what you meant or to read from resource file

Reply:
I was thinking resource file to keep it out of the code maybe?

Reply:
Sure. I will do it as a separate PR which will include moving all scripts to resources